js/src/relay/WalletLinkRelay.ts

```diff
@@ -153,7 +153,7 @@ export class WalletLinkRelay extends WalletLinkRelayAbstract {
                   const sessionIdHash = this.getSessionIdHash()
                   this.walletLinkAnalytics?.sendEvent(
                     EVENTS.UNLINKED_ERROR_STATE,
-                    { sessionIdHash }
```

Comment:
nit: can move `!wasConnectedViaStandalone` check to be on line 150 to reduce nesting